Documentation/ApiOverview/SiteHandling/AccessingSiteConfiguration.rst

```diff
@@ -1,108 +1,121 @@
-.. include:: /Includes.rst.txt
-.. index:: pair: Site handling; PHP
-.. _sitehandling-php-api:
+..  include:: /Includes.rst.txt
+..  index:: pair: Site handling; PHP
+..  _sitehandling-php-api:
 
 =====================================
 PHP API: accessing site configuration
 =====================================
 
-The PHP API for Sites comes in two parts:
+The PHP API for sites comes in two parts:
 
-- Accessing the current, resolved site object
-- Finding a site object / configuration via a page or identifier
+-   Accessing the current, resolved site object
+-   Finding a site object / configuration via a page or identifier
 
-The first case is relevant when we want to access site
-configuration in the current request, for example if we want to
-know which language is currently rendered.
+The first case is relevant when we want to access the site configuration in the
+current request, for example, if we want to know which language is currently
+rendered.
 
-The second case is about accessing site configuration options
-independent of the current request but based on a page id or a
-site identifier.
+The second case is about accessing site configuration options independent of the
+current request but based on a page ID or a site identifier.
+
+Let us look at both cases in detail.
 
-Let's look at both cases in detail:
 
 Accessing the current site object
 =================================
 
-When rendering the frontend or backend, TYPO3 builds an HTTP request object
-through a `PSR-15` middleware stack and enriches that with information.
-Part of that information are the :php:`Site` and :php:`SiteLanguage` objects. Both
-objects are available as attributes on the current request object.
+When rendering frontend or backend, TYPO3 builds an HTTP request object through
+a :ref:`PSR-15 middleware stack <request-handling>` and enriches it with
+information. Part of that information are the
+:php:`\TYPO3\CMS\Core\Site\Entity\Site` and
+:php:`\TYPO3\CMS\Core\Site\Entity\SiteLanguage` objects. Both objects are
+available as :ref:`attributes <typo3-request-attributes>` in the current
+:ref:`request object <typo3-request>`.
 
 Depending on the context, there are two main ways to access them:
 
-* via the `PSR-7` HTTP ServerRequest object directly - for example in a PSR-15 middleware
-  or the admin panel
-* via :php:`$GLOBALS['TYPO3_REQUEST']` - everywhere you don't have a ServerRequest object
+-   via the PSR-7 HTTP request object directly - for example in a PSR-15
+    middleware or the :doc:`admin panel <ext_adminpanel:Index>`
```

Comment:
```suggestion
   or the :doc:`admin panel <ext_adminpanel:Index>`
```
Why the admin panel? Aren't we talking about using the site Configuration in frontend and backend rendering here?

Reply:
Good objection - adjusted the examples.